libs/dh/admin/data-access-api/src/lib/dh-admin-user-roles-management-data-access-api.store.ts

```diff
@@ -17,21 +17,27 @@
 import { Injectable } from '@angular/core';
 import { Observable, switchMap, tap, withLatestFrom } from 'rxjs';
 import { ComponentStore, tapResponse } from '@ngrx/component-store';
-
 import {
   ErrorState,
   LoadingState,
 } from '@energinet-datahub/dh/shared/data-access-api';
 import {
   MarketParticipantUserRoleHttp,
+  CreateUserRoleDto,
   EicFunction,
   UserRoleStatus,
   UserRoleDto,
 } from '@energinet-datahub/dh/shared/domain';
 
+export interface UserRoleCreate {
```

Comment:
Suggestion: remove export, as it is not used in another file

Reply:
Fixed